.github/ISSUE_TEMPLATE/01_question.md

```diff
@@ -1,12 +1,10 @@
----
-name: ❓ Ask a question
-about: Got stuck or missing something from the docs? Ask away!
----
-
-# What can we help you with?
-
-<!-- Try to explain your question with as much detail as you can provide. -->
-
-# Where would you expect to find this information?
-
-<!-- Feel free to point us where with links or even proposing new sections or pages in the documentation. -->
+blank_issues_enabled: false
+contact_links:
+- name: Klaw Community Support
+  url: https://aiven.io/community/forum/c/open-source/12
+  about: Please ask and answer questions here.
+- name: Klaw Github discussions
+  url: https://github.com/aiven/klaw/discussions
+  about: Please ask and answer questions here.
+
+The Klaw team will cross post questions and answers across both forums, the Klaw Community Support forum which is hosted and suppported by Aiven directly, has more visibility from SEO and from other interested parties and we would encourage questions to be asked there but will respond and answer questions in both github discussions and the Community Forum. 
```

Comment:
I think we could make the sentence a bit shorter (or rather split into multiple)

```
The Klaw team will cross post questions and answers across both forums. The Klaw Community Support forum, which is hosted and supported by Aiven directly, has more visibility for other interested parties, so  we would encourage questions to be asked there. We will respond and answer questions in both GitHub discussions and the Community Forum.
```